### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.base16_theme'

### ISSUE THREAD

**Issue by jsomedon** [nvllsvm/base16-shell-preview#1]
Got this error when I run it after installation.
```
$ base16-shell-preview 
Traceback (most recent call last):
  File "/usr/bin/base16-shell-preview", line 6, in <module>
    from base16_shell_preview import main
  File "/usr/lib/python3.7/site-packages/base16_shell_preview/__init__.py", line 6, in <module>
    CURRENT_THEME = os.readlink(os.path.expanduser('~/.base16_theme'))
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.base16_theme'
```

**Comment by francis-io** [comment]
I get the same.

base16-shell-preview==0.3.0
Python 2.7.15

**Comment by kpb** [comment]
UPDATE: if you update your shell's theme with `$ base16_*` (use tab completion), the file `~/.base16_theme` will be created and base16-shell-preview works fine.

+1

python 3.6.7

**Comment by andrewrabert** [comment]
Fixed in version 0.5.1 via https://github.com/nvllsvm/base16-shell-preview/commit/9f3855eb77172425dcc3ff6ca5c958bea278c4c0

base16-shell-preview now uses one of the following to find the Base16 Shell path:
- the `BASE16_SHELL` environment variable
- the `~/.base16_theme` symlink

An error will be shown if it still cannot determine the path:
```
base16-shell-preview: error: please set the BASE16_SHELL environment variable to the local repository path.
```
